lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -282,97 +398,167 @@ def parse_genbank(self, file_path, params):
+        if len(genome['cdss']) and (genome_obj.defects['cds_seq_not_matching'] /
                                     float(len(genome['cdss'])) > 0.02):
-            self.genome_warnings.append(warnings["genome_inc_translation"].format(
-                self.defects['cds_seq_not_matching'], len(genome['cdss'])))
-            self.genome_suspect = 1
+            genome_obj.genome_warnings.append(
+                warnings["genome_inc_translation"].format(
+                    genome_obj.defects['cds_seq_not_matching'], len(genome['cdss'])
+                )
+            )
+            genome_obj.genome_suspect = 1
 
-        if self.defects['bad_parent_loc']:
-            self.genome_warnings.append(
-                f"There were {self.defects['bad_parent_loc']} parent/child "
+        if genome_obj.defects['bad_parent_loc']:
+            genome_obj.genome_warnings.append(
+                f"There were {genome_obj.defects['bad_parent_loc']} parent/child "
                 "relationships that were not able to be determined. Some of "
                 "these may have splice variants that may be valid relationships."
             )
 
-        if self.defects['spoofed_genes']:
-            self.genome_warnings.append(warnings['spoofed_genome'].format(
-                self.defects['spoofed_genes']))
+        if genome_obj.defects['spoofed_genes']:
+            genome_obj.genome_warnings.append(
+                warnings['spoofed_genome'].format(genome_obj.defects['spoofed_genes'])
+            )
             genome['suspect'] = 1
 
-        if self.defects['not_trans_spliced']:
-            self.genome_warnings.append(warnings['genome_not_trans_spliced']
-                                        .format(self.defects['not_trans_spliced']))
+        if genome_obj.defects['not_trans_spliced']:
+            genome_obj.genome_warnings.append(
+                warnings['genome_not_trans_spliced'].format(
+                    genome_obj.defects['not_trans_spliced']
+                )
+            )
             genome['suspect'] = 1
 
-        if self.genome_warnings:
-            genome['warnings'] = self.genome_warnings
-        if self.genome_suspect:
+        if genome_obj.genome_warnings:
+            genome['warnings'] = genome_obj.genome_warnings
+        if genome_obj.genome_suspect:
             genome['suspect'] = 1
-        logging.info(f"Feature Counts: {genome['feature_counts']}")
-        return genome
 
-    def _save_assembly(self, genbank_file, params):
-        """Convert genbank file to fasta and sve as assembly"""
-        contigs = Bio.SeqIO.parse(genbank_file, "genbank")
-        assembly_id = f"{params['genome_name']}_assembly"
-        fasta_file = f"{self.cfg.sharedFolder}/{params['genome_name']}_assembly.fasta"
+        if params.get('genetic_code'):
+            genome['genetic_code'] = params['genetic_code']
+
+        logging.info(f"Feature Counts: {genome['feature_counts']}")
```

Comment:
Another spot a line could be logged thousands of times

Reply:
from old code.
I removed logging.info stuff from the for loop.

Reply:
The key point here is that we're updating the old code from handling 1 upload at a time to thousands. The old code may have features like this, that are fine when only dealing with 1 upload, but that could cause issues when dealing with many. Part of this work has to be dealing with features in the old code that won't work any more.